samples-azure-functions/src/test/java/com/functions/EndToEndTests.java

```diff
@@ -36,4 +38,37 @@ public void basicChain() throws InterruptedException {
         }
         assertEquals("Completed", runTimeStatus);
     }
+
+    @Order(2)
+    @Test
+    public void testRewindInstanceAPI() throws InterruptedException {
+        Response response = post(approvalWorkFlow);
+        JsonPath rewindTestJsonPath = response.jsonPath();
+
+        // Wait for the ApprovalWorkflowOrchestration to fail
+        Thread.sleep(3000);
+
+        String instanceId = rewindTestJsonPath.get("id");
+        String statusQueryGetUri = rewindTestJsonPath.get("statusQueryGetUri");
+        Response statusResponse = get(statusQueryGetUri);
+        String runTimeStatus = statusResponse.jsonPath().get("runtimeStatus");

```

Comment:
nit: "runtime" is one word.
```suggestion
        String runtimeStatus = statusResponse.jsonPath().get("runtimeStatus");

```